### PR TITLE
parse legacy slug

### PR DESCRIPTION
**Pull request by simcha-waintrob**

### CODE REVIEW

File: cpi-side/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export const legecyPages = [
```
Comment (Chasky-H) [inline cpi-side/metadata.ts:1]: legacy

File: cpi-side/helpers/slug-parser.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
+
+import EventEmitter from 'events';
+import httpMocks, { RequestMethod } from 'node-mocks-http';
```
Comment (Chasky-H) [inline cpi-side/helpers/slug-parser.ts:3]: Do you need to add this to the package.json?

File: cpi-side/helpers/slug-parser.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import { Request, Response } from 'express';
```
Comment (Chasky-H) [inline cpi-side/helpers/slug-parser.ts:4]: Why do you need this?

File: cpi-side/helpers/slug-parser.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                // no err means no routes accepted this
+                res.json({
+                    error: 'slug not supported for ' + req.path,
+                } as SlugParseResult);
+            }
+        };
+    }
+}
+
+export default LegacySlugParser;
```
Comment (Chasky-H) [inline cpi-side/helpers/slug-parser.ts:84]: don't use default exports

File: cpi-side/helpers/slug-parser.ts (hunk 3 above)
Comment (simcha-waintrob) [reply cpi-side/helpers/slug-parser.ts:4]: finalHandler use them.

**Pull request merged**